### PR TITLE
Add Marksnorman losangeles

### LINKED ISSUE #4: Create a file for marksnorman los angeles
- [x] Create a branch
- [x] Add a commit
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by ghost**
closes #4

### CODE REVIEW

File: marksnorman-losangeles.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Los Angeles
+
+### Places to Eat
+- Birdies ... fried chicken and donuts on Olympic and Olive
+- Danny's Tacos
+
+### Places to Go
+- REDCAT
```
Comment (steven-chou) [inline marksnorman-losangeles.md]: What is this?

Comment (ghost) [reply marksnorman-losangeles.md]: It's a theater in downtown Los Angeles.

File: marksnorman-losangeles.md [hunk 2]
```diff
@@ -3,6 +3,11 @@
 ### Places to Eat
 - Birdies ... fried chicken and donuts on Olympic and Olive
 - Danny's Tacos
+- Forage ... Silverlake
+- Republic of Pies
```
Comment (steven-chou) [inline marksnorman-losangeles.md]: 👍

**Review by allthedoll — APPROVED** [review]
OMG 🌮 🌮 🌮

**Pull request merged**